src/main/java/de/szut/lf8_project/common/ValueTypeStringConverter.java

```diff
@@ -0,0 +1,14 @@
+package de.szut.lf8_project.common;
```

Comment:
die klasse kann weg, wenn wir den mapper statt hibnerate-converter nehmen

Reply:
Zählt das selbe dann nicht auch für den ValueTypeSerializer, da wir ja dann wohl DTO nutzen werden?

Reply:
da wir hier nur einen Serializer haben werden (auch in Zukunft) überwiegt für mich hier der Vorteil im Gegensatz zu n Mappen, wobei N die Anzahl der DTOs ist